src/utils/SortUtil.ts

```diff
@@ -17,5 +17,11 @@ export const sortBy =
       return -1; // Place undefined at the end
     }
 
-    return fieldA > fieldB ? order : -order;
+    // Check if both values are strings
+    if (typeof fieldA === "string" && typeof fieldB === "string") {
+      return fieldA.localeCompare(fieldB) * order;
```

Comment:
please extends test cases for string comparison with lower/upper case mixed cases in order to have a test to check comparison, maybe reporting the same names for psps that originated the bug

Reply:
done here c1eac42055f0c707986e064e4667bdfe2f41a815